doc/api/http2.md

```diff
@@ -3010,6 +3010,11 @@ properties.
   meaningful if sent by the server. Once the `enableConnectProtocol` setting
   has been enabled for a given `Http2Session`, it cannot be disabled.
   **Default:** `false`.
+* `customSettings` {object} Specifies additional settings, yet not implemented
+  in node and the underlying libraries. The key of the object define the 
+  numeric ids of the  settings and the values the numeric value of the settings.
+  This property is not supported for remote and local settings. It is only
+  supported for sending SETTINGS.
```

Comment:
This needs to be explained in greater detail.

Reply:
> This needs to be explained in greater detail.

So more like this:
```
The key of the object defines the numeric value of the settings type (as defined in the "HTTP/2 SETTINGS" registry established by [RFC7540]) and the values the actual numeric value of the settings.
 It is only supported for sending SETTINGS.
 Custom setiings are not supported for the functions retrieving remote and local settings as nghttp2 does not pass unknown HTTP/2 settings to node.js.
```

Reply:
yes